src/components/common/EntityInfo.vue

```diff
@@ -2,23 +2,25 @@
   <div id="entity-info">
     <v-container>
       <v-row no-gutters class="pt-3 pb-3">
-        <v-col cols="12" md="9">
+        <v-col cols="12" md="9" class="pr-5">
```

Comment:
I move the gutter padding to the right of this column so that, if the columns wrap, the second column isn't offset on the left side.